src/sensors/distance/ultrasound/ping/SR04.cpp

```diff
@@ -2,7 +2,7 @@
 #include "../../../../utilities/Utilities.hpp"
 namespace
 {
-const float kTimeToTravelOneCmAndBack   = 29.15F * 2.0F; // In microseconds
+const float kTimeToTravelOneCmAndBack   = 29.15f * 2.0f; // In microseconds
```

Comment:
:warning: **readability-uppercase-literal-suffix** :warning:
floating point literal has suffix 'f', which is not uppercase
```suggestion
const float kTimeToTravelOneCmAndBack   = F```